src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/cdi/ScopeDeclarationQuickFix.java

```diff
@@ -60,10 +65,7 @@ public List<? extends CodeAction> getCodeActions(JavaCodeActionContext context,
                 List<String> resultingAnnotations = new ArrayList<>(fqAnnotations);
                 resultingAnnotations.remove(annotation);
                 // For each list we will create one code action in its own context
```

Comment:
Should delete or change this comment since this code no longer copies the context.

Reply:
Removed the comment